docs/content/docs/connectors/table/kinesis.md

```diff
@@ -352,6 +352,14 @@ Connector Options
       <td>Integer</td>
       <td>Maximum number of allowed concurrent requests for the EFO client. See <a href="#enhanced-fan-out">Enhanced Fan-Out</a> for details.</td>
     </tr>
+    <tr>
+      <td><h5>scan.stream.shardassigner</h5></td>
+      <td>optional</td>
+      <td>no</td>
+      <td style="word-wrap: break-word;">DEFAULT</td>
+      <td>String</td>
+      <td>The shard assigner used to map shards to Flink sub tasks (DEFAULT|UNIFORM).</td></td>
+    </tr>
```

Comment:
Can you also update the CN docs, they should be English until they are translated. Unless you speak Chinese?!